src/cli/get-manifest.js

```diff
@@ -13,15 +13,19 @@ const { unionMerge, readManifestJSONFile, readImportedManifestFile, hasManifest
   const file = 'manifest';
   let manifest = {};
 
-  // look for a manifest files
+  // look for a manifest JSON
   const manifestJSON = readManifestJSONFile(cwd, `${file}.json`);
-  const manifestJS = readImportedManifestFile(cwd, `${file}.js`);
+  
+  // look for manifest.js
+  // stringify and parses the JSON in order to ensure that objects with .toJSON() functions
+  // resolve properly. This is a known behavior for CustomType
```

Comment:
nit: We actually do something along these lines in a bunch of places — parameters, workflows, steps, types. I'm surprised we didn't run into this elsewhere honestly!